components/core/checkbox.js

```diff
@@ -1,6 +1,25 @@
 import React from 'react'
+import Div from 'components/core/div'
+import Label from 'components/core/label'
 import styled from 'styled-components'
 
-const Checkbox = styled.input``
+const CheckboxContainer = styled(Div)`
+  align-items: center;
+`
 
-export default props => <Checkbox type="checkbox" {...props} />
+const Checkbox = styled.input`
+  margin: 0 ${props => props.theme.spaces.small}px;
+`
+
+const CheckboxLabel = styled(Label)`
+  color: ${props => props.theme.colors.white};
+  margin-right: ${props => props.theme.spaces.small}px;
+  font-size: ${props => props.theme.fontSizes.small}px;
+`
+
+export default props => (
+  <CheckboxContainer className={props.className}>
+    <Checkbox type="checkbox" checked={props.checked} id={props.id} onChange={event => props.onChange(event.target.checked)} />
+    <CheckboxLabel htmlFor={props.id}>{props.text}</CheckboxLabel>
```

Comment:
this component is legit, but it's not a checkbox anymore. It's a `<LabelledCheckbox>` - I think we should split them up.